### DISCUSSION TITLE
Email Settings in Admin panel

### DISCUSSION THREAD

**Discussion by tvld** [directus/directus#2773] [Feature Requests]
We change our email passwords at least monthly. Right now it seems that we need to rebuild the app with a new config.php each time?!
Would it not be easier to be able to set (and test ;-) ) email settings in admin panel? Our CRM uses this:

![Screenshot from 2020-07-02 09-22-52](https://user-images.githubusercontent.com/986182/86329391-623b2f80-bc46-11ea-8731-a92921e5dc8a.png)

**Comment by rijkvanzanten** [comment]
The question of _what to store where_ is a hard one to answer once and for all. The database settings obviously can't be in the database, but other than that, it's a bit up in the air.

based on:

> Right now it seems that we need to rebuild the app

I'm assuming you're running it in some sort of containerized deployment structure? Would having this in env vars help?

**Comment by tvld** [comment]
Indeed it was about Docker and we do now all with the env vars, so that works :-)  That said, it would be great if to keep the _only_ env's there the database settings.

Why not use just _global settings_ table ?  It works nice and the admin has good overview. Plus as added bonus, we could use api if needed to update, which is especially convenient if we need to fire up a new server...

Maybe we would need some menu structure one day if there will be a lot of settings to keep the page overview structured, but for now I think it will work just fine?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
